### PR TITLE
Add navbar to landing page

### PR DESCRIPTION
**Pull request by maxpinheiro**

### CODE REVIEW

File: src/Navbar.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+export default function Navbar() {
+    return (
+        <nav id="navbar-container" className="flex justify-between bg-teal-900 py-3 px-2">
```
Comment (krisharma) [inline src/Navbar.js:5]: Can you verify this is the same color as on the design?
Here's a link on custom colors for Tailwind: https://tailwindcss.com/docs/customizing-colors 
Design Reference: https://xd.adobe.com/view/3da03753-f1e6-4bac-82db-20f1fcb93155-68f3/specs/

File: src/Navbar.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+                        <a className="" href="/modules">
+                            <p className="text-xs font-bold text-white uppercase">
+                                Modules
+                            </p>
+                        </a>
+                    </li>
+                    <li id="nav-item" className="mx-8 border-b-2 border-transparent hover:border-teal-300 duration-300 ease-in-out">
+                        <a className="" href="/about">
+                            <p className="text-xs font-bold text-white uppercase">
+                                About
```
Comment (krisharma) [inline src/Navbar.js:30]: Similar to the color, can we verify that the font family is the same?
Link on adding Google Fonts to Tailwind: https://scottw.com/blog/google-font-tailwind/

File: src/App.js [hunk 3]
```diff
@@ -1,18 +1,20 @@
...
-          Sandbox A-Eye Project Landing Page
-        </p>
-      </header>
-    </div>
+      <div className="App">
+          <div className="App-header">
+              <Navbar />
+          </div>
+
+          <img src={logo} className="App-logo" alt="logo" />
```
Comment (krisharma) [inline src/App.js:13]: Note that this is no longer centered - it'll be removed but just wanted to draw your attention to it

**Review by krisharma — CHANGES_REQUESTED** [review]
Overall looks great! Just take a look at the two minor comments and then it'll be good to merge :)
Thanks a ton again for your help in getting Tailwind set up

File: src/Navbar.js (hunk 1 above)
Comment (maxpinheiro) [reply src/Navbar.js:5]: Oops, I meant to leave a comment about this; I haven't figure out yet how to customize colors with Tailwind, so I just used an approximate color as a placeholder. I'll get started on customizing the colors and font :)

File: src/Navbar.js (hunk 2 above)
Comment (maxpinheiro) [reply src/Navbar.js:30]: Same as above ^ Should I make these changes before merging this commit or can it be a second commit?

**Review by krisharma — APPROVED** [review]
LGTM! Thanks Max 👍

**Pull request merged**